### DISCUSSION TITLE
Dialog fix top

### DISCUSSION THREAD

**Discussion by maicWorkGithub** [quasarframework/quasar#12098] [General - Components / Directives / etc] [answered]
online example: https://codepen.io/maicworkgithub/full/qBPwzwj

The example shows that when there are tabs in the dialog and the heights of the panels of the tabs are different, the dialog will flicker.
I want to make a dialog with a fixed top value (top value is not equal to 0), and the height of the dialog will change with the height of the content.
Is there any way to do this currently?
At present, it seems that I just need to remove the fixed-full class on the .q-dialog__inner element and add the top style value I want to q-card.

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/gOGyVBV?editors=1010
